### ISSUE TITLE
Cookie pop up: UK law?

### ISSUE THREAD

**Issue by ijmccallum** [WikiLogic/foundation#5]
Figure out the requirements & if needed - code it.

**Comment by ijmccallum** [comment]
@douglassophies - you asked if this was something you might help with? Yep! Definetly! I think it will probably just be a formality, but I think we might need to have some note on the site that says "we use cookies, hope that's cool!". We don't, but I be WordPress does which might mean we are required to have a note.

**Comment by DouglasUT2018** [comment]
Ok added to my WL charity to do list. Would it make sense to combine the 
Blog posts ideas with the general WL charity? Means there is only 2 
lists instead of three to work of, i.e. General charity work and 
programming tickets.

**Comment by ijmccallum** [comment]
Yeah - my main thing was turning the notes from the project lists into issues so we can have a conversation like this one!

**Comment by DouglasUT2018** [comment]
If i understand you right, can we not do that with the main WL charity list?

**Comment by ijmccallum** [comment]
Think I'm confused, this is my understanding of the github project system:
Notes in the WLCharity project (https://github.com/orgs/WikiLogic/projects/1) can be turned into issues, but to be an issue they have to be related to one of the repos.

So this one can actually be added to your todo list! Watch...

**Comment by DouglasUT2018** [comment]
I can see how the trail of comments on a ticket are handy. In trello you can keep adding comments but it seems you can only edit the note here and that has a character limit. Otherwise i would say that we dont need an issue ticket and a todo ticket. For example the cookies issue does not really interfere with the foundation website so it would not go as an issue there. It more in the law and business side so we put in the WLCharity kanban board. But the kanban board limitations means i think we can just do as we like and not try to set a formal process yet ;)

**Comment by DouglasUT2018** [comment]
Everything you need to know at this link. (See "Use on EUROPA heading" in particular) http://ec.europa.eu/ipg/basics/legal/cookies/index_en.htm

**Comment by ijmccallum** [comment]
Opening again! This particular one will affect the foundation site if I have to add a note to the theme. Had a skim through that link and I think we're borderline. Need to look through the cookies that are being used & possibly set up some markup & styling.
